admin-dev/themes/new-theme/js/pages/product/services/paginated-combinations-service.ts

```diff
@@ -28,6 +28,9 @@ import PaginationServiceType from '@PSTypes/services';
 
 const {$} = window;
 
+/**
+ * @todo: rename to FilterableCombinationsService?
+ */
 export default class PaginatedCombinationsService implements PaginationServiceType {
```

Comment:
Not sure the renaming is necessary, the filter feature is quite common in pagination so even it doesn't necessarily handles it (depends on use cases) it's kind of implied that it could

Reply:
Actually it would even make sense to add `setFilters` and `setOrderBy` (at least) in the `PaginationServiceType` (not all paginated list will handle them but they can still implement them as empty methods so it's fine)

I don't know if we should also add the getters as well `getFilters` `getOrderBy` but why not